### PR TITLE
wishlistpg

### PR DESCRIPTION
**Pull request by aurobindoGupta**
image

### CODE REVIEW

**Review by hiteshverma27 — APPROVED** [review]
Looks good

File: components/cartPg/index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        <li class="navLinksBtn">
+          <button class="btn btn-secondary">Login</button>
+        </li>
+        <li class="navLinksBtn" onclick="window.location.href=''">
+          <div class="ic-badge container">
+            <i class="fa fa-heart ic"></i>
+            <div class="iconBadge">3</div>
+          </div>
+        </li>
+        <li class="navLinksBtn" onclick="window.location.href=''">
```
Comment (Aditi0505) [inline components/cartPg/index.html:43]: Remove onClick if not needed.

File: components/wishListPg/index.html [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+      <div class="navBar-searchBar">
+        <form class="searchBar">
+          <input class="searchBar-input" placeholder="Search" />
+        </form>
+      </div>
+      <ul class="navLinks">
+        <li class="navLinksBtn">
+          <button class="btn btn-secondary">Login</button>
+        </li>
+        <li class="navLinksBtn" onclick="window.location.href=''">
```
Comment (Aditi0505) [inline components/wishListPg/index.html:37]: Can remove onClick if not needed.

File: components/wishListPg/index.html [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+          <div class="ic-badge container">
+            <i class="fa fa-heart ic"></i>
+            <div class="iconBadge">3</div>
+          </div>
+        </li>
+        <li class="navLinksBtn" onclick="window.location.href=''">
```
Comment (Aditi0505) [inline components/wishListPg/index.html:43]: Same as above.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Rest looks good.

File: components/wishListPg/index.html (hunk 3 above)
Comment (aurobindoGupta) [reply components/wishListPg/index.html:43]: removed in separate branch

**Pull request merged**